### PR TITLE
feat: add travis job

### PR DESCRIPTION
**Pull request by dtopuzov**
- Add travis.yml
- Redirect brew install log to /dev/null (to reduce verbosity, this is also important for travis job)
- Install Open JDK 8 instead of Oracle JDK 8
- Reduce verbosity on Android SDK update (progress bar in console breaks travis ci)
- Fix emulator creation (echo no instead of echo y)

### CODE REVIEW

File: native-script.rb [hunk 1]
```diff
@@ -108,71 +107,66 @@ def install_environment_variable(name, value)
...
 error_msg = "There seem to be some problems with the Android configuration"
 
 sdk_manager = File.join(ENV["ANDROID_HOME"], "tools", "bin", "sdkmanager")
-execute("echo y | #{sdk_manager} \"platform-tools\"", error_msg)
-execute("echo y | #{sdk_manager} \"tools\"", error_msg)
-execute("echo y | #{sdk_manager} \"build-tools;28.0.3\"", error_msg)
-execute("echo y | #{sdk_manager} \"platforms;android-28\"", error_msg)
-execute("echo y | #{sdk_manager} \"extras;android;m2repository\"", error_msg)
-execute("echo y | #{sdk_manager} \"extras;google;m2repository\"", error_msg)
+execute("echo y | #{sdk_manager} \"tools\" | grep -v = || true", error_msg)
```
Comment (rosen-vladimirov) [inline native-script.rb]: again we hide output from the user, which is not expected

File: native-script.rb [hunk 2]
```diff
@@ -108,71 +107,66 @@ def install_environment_variable(name, value)
...
+    execute("echo y | #{sdk_manager} \"system-images;android-28;google_apis;x86\" | grep -v = || true", "Failed to download Android emulator system image.")
   end
 end
 
 puts "Do you want to create Android emulator? (y/n)"
 if $silentMode || gets.chomp.downcase == "y"
   error_msg = "Failed to create Android emulator."
   avd_manager = File.join(ENV["ANDROID_HOME"], "tools", "bin", "avdmanager")
-  execute("echo y | #{avd_manager} create avd -n Emulator-Api28-Google -k  \"system-images;android-28;google_apis;x86\" -b google_apis/x86 -c 265M -f", error_msg)
+  execute("echo no | #{avd_manager} create avd -n Emulator-Api28-Google -k  \"system-images;android-28;google_apis;x86\" -b google_apis/x86 -c 265M -f", error_msg)
```
Comment (rosen-vladimirov) [inline native-script.rb]: why is this `y` changed to `no` - can you please add a comment

File: native-script.rb (hunk 1 above)
Comment (rosen-vladimirov) [reply native-script.rb]: Again, we can use NS_SETUP_HIDE_OUTPUT env variable

**Comment by etabakov** [comment]
@cla-bot check

**Comment by etabakov** [comment]
@cla-bot check

**Pull request merged**